arcgispro/src/coregister/esri/toolboxes/3D Data Co-Registration.pyt

```diff
@@ -549,10 +554,42 @@ class Register_MultiType(object):
             if parameters[5].value <= parameters[6].value:
                 parameters[5].setErrorMessage("Strong filter size must be larger than weak filter size")
                 parameters[6].setErrorMessage("Weak filter size must be smaller than large filter size")
-        return
+        
+        #Check if input DEMs have equal X and Y cell size values
+
+        #first make sure both params are input
+        if parameters[0].value and parameters[1].value:
+            #get path split into head and tail
+            fnd_list = os.path.split(parameters[0].valueAsText)
+            aoi_list = os.path.split(parameters[1].valueAsText)
+            fnd_full_path = os.fsdecode(f"{parameters[0].valueAsText}").replace(os.sep, "/")
+            aoi_full_path = os.fsdecode(f"{parameters[1].valueAsText}").replace(os.sep, "/")
+
+            inputs_list =[fnd_full_path, aoi_full_path]
+            #check for both FND and AOI
+            for input_file in inputs_list:
+                #analysis can only be done with raster/DEM inpu
+                if input_file[-3:] == 'tif' or input_file[-3:] == 'iff':
```

Comment:
if we're wanting to check for the extension; instead of checking the last 3 characters of a string, we should check the extension a bit more directly, we can do the following instead

```python
if os.path.splitext(input_file)[-1] in {'.tif', '.tiff'}:
   ...
```

This relays the intent a bit more directly that we are checking for the filename extension, also makes it easier to add other extensions we want to check against, speaking of which, besides `tif` and `tiff` are there other extensions we want to check against?